.github/actions/setup-environment/action.yaml

```diff
@@ -26,6 +26,22 @@ runs:
       # Setting the default version also lets WSL finish updating.
       wsl --set-default-version 2
 
+  - name: "Windows: Install yq"
+    if: runner.os == 'Windows'
+    shell: bash
+    run: |
+      set -o xtrace
+      bindir="$HOME/bin"
+      if [[ ! "$PATH" =~ "$bindir" ]]; then
+        bindir=/usr/bin
```

Comment:
`/usr/local/bin` might be a better fallback?

Reply:
Might be, but then I have to check if `/usr/local/bin` is on the `PATH` on GitHub runners.

Given that the existing code was working as-is, I think I don't want to touch it.